doc/user-guide/data-structures.rst

```diff
@@ -800,6 +800,7 @@ included by default unless you exclude them with the ``inherited`` flag:
 
     dt2["/weather/temperature"].to_dataset(inherited=False)
 
+For more examples and further discussion see LINK
```

Comment:
I want to use this link but I can't seem to get it to work properly

`:ref:`Alignment and Coordinate Inheritance <userguide.hierarchical-data.alignment-and-coordinate-inheritance>`

Reply:
fixed in https://github.com/pydata/xarray/pull/9501/commits/82a70a00f7f0fbdec45522abf28d189cad764b6b